webapp/tests/forms/steps/lotse/test_pauschbetrag.py

```diff
@@ -0,0 +1,214 @@
+from app.forms.steps.lotse.pauschbetrag import calculate_pauschbetrag
+
+
+class TestCalculatePauschbetrag:
+
+    def test_if_no_merkzeichen_or_pflegegrad_set_then_return_correct_value_for_beh_grad(self):
+        input_output_pairs = [
+            (20, 384),
+            (30, 620),
+            (40, 860),
+            (50, 1140),
+            (60, 1440),
+            (70, 1780),
+            (80, 2120),
+            (90, 2460),
+            (100, 2840),
+        ]
+
+        params = {
+            'pflegegrad': False,
+            'merkzeichen_bl': False,
+            'merkzeichen_tbl': False,
+            'merkzeichen_h': False,
+            'merkzeichen_ag': False,
+            'merkzeichen_g': False,
+        }
+
+        for beh_grad, expected_result in input_output_pairs:
+
+            calculated_pauschbetrag = calculate_pauschbetrag(**params, beh_grad=beh_grad)
+
+            assert calculated_pauschbetrag == expected_result
```

Comment:
So nice taking the params!
(I just dislike the empty lines 29 and 31 - thought I'd mention it if you maaayyybe didn't add them on purpose. 🙈 )